compiler/circle-mpqsolver/src/core/SolverHooks.h

```diff
@@ -34,10 +34,11 @@ class SolverHooks
   /**
    * @brief called on the start of iterative search
    * @param model_path path of original float model to quantize
-   * @param q8error error of Q8 quantization
-   * @param q16error error of Q16 quantization
+   * @param q8error error of Q8 quantization (if nullptr, then not applicable)
+   * @param q16error error of Q16 quantization (if nullptr, then not applicable)
```

Comment:
can you use `NaN` ?

Reply:
@seanshpark
It seems that NaN can be used instead of nullptr. Thank you.